pkgs/os-specific/linux/xp-pen-drivers/deco-01-v2/default.nix

```diff
@@ -24,8 +24,7 @@ stdenv.mkDerivation rec {
 
   src = fetchzip {
     url = "https://www.xp-pen.com/download/file/id/1936/pid/440/ext/gz.html#.tar.gz";
-    name = "xp-pen-deco-01-v2-driver-${version}.tar.gz";
```

Comment:
The name is used here to make the store path name nicer. The default one will be something like `gz.html#.tar.gz`

Reply:
Ok! Thank you for the explaination.